src/VisualStudio/Core/Test/ServicesVisualStudioTest.vbproj

```diff
@@ -146,16 +143,16 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="WindowsBase" />
   </ItemGroup>
-  <Target Name="AddShellImmutable15" Condition="'$(VisualStudioVersion)' == '15.0'" BeforeTargets="ResolveAssemblyReferences">
-    <!-- we want this to only be added for Dev15 builds, but the project system doesn't like us having conditional
+  <!--<Target Name="AddShellImmutable15" Condition="'$(VisualStudioVersion)' == '15.0'" BeforeTargets="ResolveAssemblyReferences">
```

Comment:
This allows it to build, but doesn't seem right to me.

Maybe the solution here is for NuGet to pull in all of Shell.14, Shell.15, and Shell.Immutable.15, but then have these targets *remove* the ones for the version you *aren't* building against?